packages/cozy-konnector-libs/scripts/visualizer/index.js

```diff
@@ -24,6 +24,7 @@ const generate = async options => {
 }
 
 process.on('unhandledRejection', x => {
+  // eslint-disable-next-line no-console
```

Comment:
don't like this rule for one line

if we use console.log, just let's put the rule in the config file

Reply:
I actually want to enforce this rule in cozy-konnector-libs. But this file is a script to generate a visualization of links between doctypes and is not run by the stack, then we don't need to use log here.